test/services/twilio-api/twilio-client.test.js

```diff
@@ -265,6 +265,200 @@ describe('services', () => {
           expect(options.PageSize).to.be.undefined;
         });
       });
+
+      describe('regional and edge support', () => {
+        let regionTest = test
```

Comment:
Don't like reassigning this between tests. IDEs may run this tests in different order and I _think_ this could cause a breakage.